Source/NSTableView.m

```diff
@@ -2866,7 +2867,7 @@ - (void) selectRowIndexes: (NSIndexSet *)indexes byExtendingSelection: (BOOL)ext
       if ([indexes lastIndex] >= _numberOfRows)
         {
 	  [NSException raise: NSInvalidArgumentException
-		       format: @"Row index out of table in selectRow"];
+                      format: @"Row index out of table in selectRow"];
```

Comment:
This seems to have changed since last review, but is still not aligned on `:`.

Reply:
I will try to do better :-)

Reply:
Done